File: src/main/java/de/sean/splugin/util/SLockUtil.java
package de.sean.splugin.util;

import org.bukkit.block.Block;

import java.util.HashMap;

public class SLockUtil {
    public static final String OWNER_ATTRIBUTE = "splugin_owner";
    public static final String LOCK_ATTRIBUTE = "splugin_lock";
    public static final String REDSTONE_ATTRIBUTE = "splugin_lock_redstone";

    public static final HashMap<String, Block> lock = new HashMap<>();
}

File: src/main/kotlin/de/sean/splugin/util/SLockUtil.kt
package de.sean.splugin.util

import org.bukkit.block.Block
import java.util.HashMap

object SLockUtil {
    const val OWNER_ATTRIBUTE = "splugin_owner"
    const val LOCK_ATTRIBUTE = "splugin_lock"
    const val REDSTONE_ATTRIBUTE = "splugin_lock_redstone"
    val lock = HashMap<String, Block>()
}
